src/utils.ts

```diff
@@ -175,3 +177,35 @@ export const calculateRpStatus = (
 
   return calculatedStatus;
 };
+
+export const getAdditionalInfo = (test: TestCase): TestAdditionalInfo => {
+  const initialValue: TestAdditionalInfo = {
+    attributes: [],
+    description: '',
+    testCaseId: '',
+    status: '',
+  };
+
+  return test.results.reduce<TestAdditionalInfo>(
+    (additionalInfo, { attachments = [] }) =>
+      Object.assign(
+        additionalInfo,
+        attachments.reduce<TestAdditionalInfo>((acc, { name, body }) => {
+          if (name in RPTestInfo) {
+            try {
+              const value = body.toString();
+
+              return Object.assign(acc, {
+                [name]: name === RPTestInfo.attributes ? JSON.parse(value) : value,
+              });
+            } catch (error: unknown) {
```

Comment:
Can we use `Error` type right here?

Reply:
TS enforces you to use an unknown type here, that is why we can not use error:Error here